android/KMEA/app/src/main/java/com/tavultesoft/kmea/util/KMLog.java

```diff
@@ -17,6 +17,21 @@
 public final class KMLog {
   private static final String TAG = "KMLog";
 
+  /**
+   * Utility to determine if Sentry library is available in an app
+   * @return boolean
+   */
+  public static boolean sentryLibraryExists() {
+    boolean result = false;
+    try {
+      Class.forName("io.sentry.Sentry");
+      return true;
+    } catch (ClassNotFoundException e) {
+      // Intentionally not sending to Sentry because 3rd party apps may not include Sentry library
+      return false;
+    }
+  }
```

Comment:
Can we set this up to run only once and store the result for future executions? Exceptions in Java are [expensive](https://stackoverflow.com/questions/299068/what-are-the-effects-of-exceptions-on-performance-in-java).

Reply:
OK, I've refactored this to DependencyUtil. But KMLog won't have context available to store the result in a SharedPreference so it'll still go through the Exception once before storing it statically.

Reply:
hmm, I think just an in-memory value is fine. Don't think we should store it long-term. One exception on first use is acceptable.